.github/workflows/test.yaml

```diff
@@ -0,0 +1,35 @@
+name: test
+
+on:
+  push:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Install Buf
+        uses: bufbuild/buf-setup-action@v0.5.0
+      - name: Lint Protobufs
+        uses: bufbuild/buf-lint-action@v1
+      - name: Check Breaking Changes
+        if: github.ref != 'refs/heads/main'
+        uses: bufbuild/buf-breaking-action@v1
+        continue-on-error: true
```

Comment:
Ignoring failures here for now for two reasons:
1. There are no protos in `main` at the moment
2. We'll probably intentional introduce breaking changes, but this could catch accidental changes

If it proves to be unhelpful  we can just remove the step.